modules/features/endofyear/src/main/java/au/com/shiftyjelly/pocketcasts/endofyear/components/StoryText.kt

```diff
@@ -21,14 +25,18 @@ fun StoryPrimaryText(
     color: Color,
     modifier: Modifier = Modifier,
 ) {
-    TextH20(
+    TextH10(
         text = text,
         textAlign = TextAlign.Center,
         color = color,
-        disableScale = disableScale(),
+        disableScale = true,
+        fontFamily = FontFamily(listOf(Font(UR.font.dm_sans))),
+        fontWeight = FontWeight.W600,
+        fontSize = 25.sp,
+        lineHeight = 31.sp,
```

Comment:
I have used a smaller font size and line height than the ones given in `Figma` (`size=30`, `lineHeight=33`) as the text took much of the screen space, particularly on the tablet. 

I can provide separate font sizes for phones and tablets if they do not look good.

Reply:
Looks good to me.